libs/langchain/langchain/utilities/clickup.py

```diff
@@ -14,50 +14,44 @@ class ClickupAPIWrapper(BaseModel):
     redirect_url: Optional[str] = None
     code: Optional[str] = None
     access_token: Optional[str] = None
-
+    url: Optional[str] = "https://api.clickup.com/api/v2/oauth/token"
+    team_id: Optional[str] = None
+ 
     class Config:
         """Configuration for this pydantic object."""
 
         extra = Extra.forbid
+    
+    def post_init(self) -> None:
```

Comment:
Let's remove this if it doesn't work?  Will just handle it in validation instead